### PR TITLE
minor updates for docs

### PR DESCRIPTION
**Pull request by csantanapr**
Making minor changes to docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,14 @@ These steps were tested on Mac OS X El Capitan, Ubuntu 14.04.3 LTS and may work
 * [Vagrant on Mac, Windows PC, or GNU/Linux](tools/vagrant/custom/README.md)
 * [OpenWhisk on Ubuntu natively](tools/ubuntu-setup/README.md)
 
+## Configure CLI
+
+The OpenWhisk CLI is located in `/bin/wsk`
+If not already configured, then it should be configure with the `namespace` "guest" and `auth` key from `ansible/files/auth.guest`
+For example run the following command
+```
+./bin/wsk --namespace guest --auth `cat ansible/files/auth.guest`
```
Comment (rabbah) [inline README.md]: missing `property set`.

File: docs/README.md [hunk 2]
```diff
@@ -84,4 +90,4 @@ You can find additional information about OpenWhisk in the following topics:
 * [Entity names](./reference.md#openwhisk-entities)
 * [Action semantics](./reference.md#action-semantics)
 * [Limits](./reference.md#system-limits)
-* [REST API](./reference.md#rest-api)
+* [REST API](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/openwhisk/openwhisk/master/core/controller/src/main/resources/whiskswagger.json)
```
Comment (rabbah) [inline docs/README.md]: really? this skips some of the useful bits in reference.md.

File: docs/mobile_sdk.md [hunk 3]
```diff
@@ -40,23 +40,23 @@ You must then add OpenWhisk.framework to the embedded frameworks in your Xcode p
 
 ### Installing from source code
 
-Source code is available at https://github.com/openwhisk/openwhisk-client-swift.git. Open project using the OpenWhisk.xcodeproj file in Xcode.  The project contains two schemes "OpenWhisk" and "OpenWhiskWatch" targetted for iOS and watchOS 2, respectively.  Build the project for the targets you need and add the resulting frameworks to your app (usually in ~/Library/Developer/Xcode/DerivedData/your app name).
+Source code is available at https://github.com/openwhisk/openwhisk-client-swift.git. Open project using the OpenWhisk.xcodeproj file in Xcode. The project contains two schemes "OpenWhisk" and "OpenWhiskWatch" targetted for iOS and watchOS 2, respectively. Build the project for the targets you need and add the resulting frameworks to your app (usually in ~/Library/Developer/Xcode/DerivedData/your app name).
```
Comment (rabbah) [inline docs/mobile_sdk.md]: Open _the_ (missing).

File: README.md (hunk 1 above)
Comment (csantanapr) [inline README.md]: good catch

File: docs/README.md (hunk 2 above)
Comment (csantanapr) [inline docs/README.md]: hum, I was not %100 sure about this one, I will leave pointing to reference.

**Comment by csantanapr** [comment]
@rabbah updated, adjust and merge please.

**Comment by rabbah** [comment]
Rebased and merged as https://github.com/openwhisk/openwhisk/commit/99d3e9ab82756631285072ad551398fa6b034e21. One small tweak reviewed with @csantanapr.

**Pull request closed without merging**